xc7/utils/prjxray_form_channels.py

```diff
@@ -207,38 +203,36 @@ def import_grid(db, grid, conn):
         # tile: pkey name tile_type_pkey grid_x grid_y
         c.execute(
             """
-INSERT INTO tile(name, tile_type_pkey, grid_x, grid_y)
+INSERT INTO phy_tile(name, tile_type_pkey, grid_x, grid_y)
 VALUES
   (?, ?, ?, ?)""",
             (tile, tile_types[gridinfo.tile_type], loc.grid_x, loc.grid_y)
         )
 
-    build_other_indicies(c)
     c.connection.commit()
 
 
-def import_nodes(db, grid, conn):
+def import_nodes(db, conn):
     # Some nodes are just 1 wire, so start by enumerating all wires.
 
     c = conn.cursor()
+    c1 = conn.cursor()
     c2 = conn.cursor()
     c2.execute("""BEGIN EXCLUSIVE TRANSACTION;""")
 
     tiles = {}
     tile_wire_map = {}
     wires = {}
-    for tile in progressbar.progressbar(grid.tiles()):
-        gridinfo = grid.gridinfo_at_tilename(tile)
-        tile_type = db.get_tile_type(gridinfo.tile_type)
 
-        c.execute(
-            """SELECT pkey, tile_type_pkey FROM tile WHERE name = ?;""",
-            (tile, )
-        )
-        tile_pkey, tile_type_pkey = c.fetchone()
-        tiles[tile] = (tile_pkey, tile_type_pkey)
+    for tile_name, tile_pkey, tile_type_pkey in progressbar.progressbar(
+            c1.execute("SELECT name, pkey, tile_type_pkey FROM tile")):
```

Comment:
This should be phy_tile?

Reply:
Actually when there is no tile split yet it does not matter. I still haven't figured out though what to do with all connections when CLBs are split.

Reply:
My comment absolutely stands.  Connections using the prjxray DB will always use the physical grid, never the VPR grid.

Reply:
As a concrete example, on line 235, the tile_type must be the prjxray tile type name, not the VPR tile type name.  SLICEL is not a tile type, CLBLL_L is.

Reply:
Ok, you are right. This has to be a phy_tile. In this function tile location isn't concerned, just the type.